### PR TITLE
2661 - Translation system 

### LINKED ISSUE #2661: Translation system
- [ ] put translations online system 
- [ ] integrate online system through prs 
- [ ] document main features

### PR DESCRIPTION
**Pull request by yaguzmang**
Platform translations working in new format:

https://github.com/user-attachments/assets/cc87ba18-798c-439b-af77-f34615236ab3

Admin side:

https://github.com/user-attachments/assets/25d50e0e-adcf-4708-b93e-63d31359b5a7

Email:
(en)

image

(es)

image

### CODE REVIEW

File: src/i18n/i18nFactory.ts [hunk 1]
```diff
@@ -3,23 +3,23 @@ import { TFunction } from 'i18next'
...
 const translationsFiles: { [langCode: string]: any } = {
-  en: enTranslation.translation,
-  es: esTranslation.translation,
-  fr: frTranslation.translation,
-  ru: ruTranslation.translation,
-  ar: arTranslation.translation,
-  zh: zhTranslation.translation,
+  en: enTranslation,
+  es: esTranslation,
+  fr: frTranslation,
```
Comment (sorja) [inline src/i18n/i18nFactory.ts]: Minor suggestion, not sure if relevant

```suggestion
import { en } from './resources/en'
import { es} from './resources/es'
import { fr } from './resources/fr'

// @ts-ignore
const createInstance = i18next.createInstance || i18next.default.createInstance

const translationsFiles: { [langCode: string]: any } = {
  en,
  es,
  fr,
```

Comment (sorja) [reply src/i18n/i18nFactory.ts]: Do we have type for langCode?
and can we avoid any

check also other params in file

Comment (yaguzmang) [reply src/i18n/i18nFactory.ts]: Adding it in the next PR 👍

**Comment by minotogna** [comment]
@sorja this looks go to me . For your final approval

**Review by sorja — APPROVED** [review]

**Pull request merged**